### PR TITLE
add class Range (WIP)

### PR DESCRIPTION
**Pull request by VasavanThiru**
range_ros

### CODE REVIEW

File: src/viz/Range.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import * as THREE from 'three';
+
+import Core from '../core';
+import {DEFAULT_OPTIONS_POLYGON, MESSAGE_TYPE_RANGE} from '../utils/constants';
+import {DEFAULT_COLOR_ARROW} from '../utils/constants';
+import Group from '../primitives/Group';
+import Cone from '../primitives/Cone';
+
+class Range extends Core {
+  constructor(ros, topicName, options = DEFAULT_OPTIONS_POLYGON) {
```
Comment (wolfv) [inline src/viz/Range.js]: can you create a DEFAULT_OPTIONS_RANGE?

File: src/viz/Range.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    super(ros, topicName, MESSAGE_TYPE_RANGE, options);
+    this.object = new Group();
+    this.updateOptions({
+      ...DEFAULT_OPTIONS_POLYGON,
```
Comment (wolfv) [inline src/viz/Range.js]: here too.

File: src/viz/Range.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+      ...options
+    });
+  }
+
+  updateOptions(options) {
+    super.updateOptions(options);
+    const { color, alpha } = this.options;
```
Comment (wolfv) [inline src/viz/Range.js]: what options exist in RViz?

Comment (chaitanya-deep) [reply src/viz/Range.js]: http://wiki.ros.org/rviz/DisplayTypes/Range

I guess the buffer length can be ignored for now

**Pull request merged**